rawkit/raw.py

```diff
@@ -7,6 +7,7 @@
 import random
 import string
 import tempfile
+import numpy as np
```

Comment:
We definitely don't want a hard dependency on numpy. Can we use a regular array for now, and we'll figure out a good general API to support numpy (hopefully w/o copying data to Python land first) later?